src/components/inputs/Input/index.js

```diff
@@ -216,7 +216,7 @@ class Input extends PureComponent {
                                     ? event => event.target.select()
                                     : null
                             }
-                            borderColor={this.getColor(this.props.state)}
+                            border={this.getColor(this.props.state)}
```

Comment:
What is the purpose of this `border` prop when you always pass the same value?

Reply:
You don't always pass the same value. 
There is the prop variable 'state' which can be 'success', 'warning' or 'error' and based on that the border color of the input/textarea is changing.